crowbar_framework/config/repos-cloud.yml

```diff
@@ -73,25 +73,25 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
-    suse-enterprise-storage-2_1-pool:
-      name: "SUSE-Enterprise-Storage-2.1-Pool"
+    suse-enterprise-storage-3-pool:
+      name: "SUSE-Enterprise-Storage-3.0-Pool"
```

Comment:
I cannot find the example now, but for SES 2.0 the strings contained "2.0", that's why I used here "3.0". I think also allows for increased versions easily.